grabbit/src/main/groovy/com/twcable/grabbit/client/servlets/GrabbitCleanJobRepositoryServlet.groovy

```diff
@@ -0,0 +1,27 @@
+package com.twcable.grabbit.client.servlets
+
+import com.twcable.grabbit.spring.batch.repository.services.CleanJobRepository
+import groovy.transform.CompileStatic
+import groovy.util.logging.Slf4j
+import org.apache.felix.scr.annotations.Reference
+import org.apache.felix.scr.annotations.sling.SlingServlet
+import org.apache.sling.api.SlingHttpServletRequest
+import org.apache.sling.api.SlingHttpServletResponse
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet
+
+import javax.annotation.Nonnull
+
+@Slf4j
+@CompileStatic
+@SlingServlet(methods = ['GET'], paths = ["/bin/cleanRepository"])
+class GrabbitCleanJobRepositoryServlet extends SlingSafeMethodsServlet {
+
+    @Reference
+    CleanJobRepository cleanJobRepository
+
+    @Override
+    protected void doGet( @Nonnull SlingHttpServletRequest request, @Nonnull SlingHttpServletResponse response) {
+        int hours = request.getParameter("hours").toInteger()
```

Comment:
The servlet will now explicitly check for hours parameter and if it is not present, it will send a "BAD REQUEST" response. !fixed